Distribution/Server/Users/Types.hs

```diff
@@ -28,14 +33,23 @@ newtype UserName  = UserName String
 
 data UserInfo = UserInfo {
                   userName   :: !UserName,
-                  userStatus :: !UserStatus
+                  userStatus :: !UserStatus,
+                  userTokens :: !UserTokenSet
+                } deriving (Eq, Show, Typeable)
+
+data UserInfo_v0 = UserInfo_v0 {
```

Comment:
Our typical convention is to stick the old versions down at the end of the file, so they don't get in the way of reading too much.